tests/data_context/cloud_data_context/test_include_rendered_content.py

```diff
@@ -0,0 +1,184 @@
+from typing import Type
+from unittest import mock
+
+import pandas as pd
+import pytest
+
+from great_expectations.core import (
+    ExpectationConfiguration,
+    ExpectationSuite,
+    ExpectationValidationResult,
+)
+from great_expectations.core.batch import RuntimeBatchRequest
+from great_expectations.data_context import (
+    AbstractDataContext,
+    BaseDataContext,
+    CloudDataContext,
+    DataContext,
+)
+from great_expectations.render.types import RenderedAtomicContent
+from great_expectations.validator.validator import Validator
+
+
+@pytest.mark.cloud
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "data_context_fixture_name,data_context_type",
+    [
+        # In order to leverage existing fixtures in parametrization, we provide
+        # their string names and dynamically retrieve them using pytest's built-in
+        # `request` fixture.
+        # Source: https://stackoverflow.com/a/64348247
+        pytest.param(
+            "empty_base_data_context_in_cloud_mode",
+            BaseDataContext,
+            id="BaseDataContext",
+        ),
+        pytest.param("empty_data_context_in_cloud_mode", DataContext, id="DataContext"),
+        pytest.param(
+            "empty_cloud_data_context", CloudDataContext, id="CloudDataContext"
+        ),
+    ],
+)
+def test_cloud_backed_data_context_save_expectation_suite_include_rendered_content(
+    data_context_fixture_name: str,
+    data_context_type: Type[AbstractDataContext],
+    request,
+) -> None:
+    """
+    All Cloud-backed contexts (DataContext, BaseDataContext, and CloudDataContext) should save an ExpectationSuite
+    with rendered_content by default.
+    """
+    context = request.getfixturevalue(data_context_fixture_name)
+
+    with mock.patch(
+        "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend.list_keys"
+    ), mock.patch(
+        "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend._set"
+    ):
+        expectation_suite: ExpectationSuite = context.create_expectation_suite(
+            "test_suite"
+        )
+    expectation_suite.expectations.append(
+        ExpectationConfiguration(
+            expectation_type="expect_table_row_count_to_equal", kwargs={"value": 10}
+        )
+    )
+    assert expectation_suite.expectations[0].rendered_content is None
+
+    with mock.patch(
+        "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend.list_keys"
+    ), mock.patch(
+        "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend._update"
+    ) as mock_update:
+        context.save_expectation_suite(
+            expectation_suite,
+        )
+
+        # remove dynamic great_expectations version
+        mock_update.call_args[1]["value"].pop("meta")
+
+        mock_update.assert_called_with(
+            ge_cloud_id="None",
```

Comment:
Why "None" instead of `None`?

Reply:
Because of this line: https://github.com/great-expectations/great_expectations/blob/1cbf5147c87d045a93eab21ebeda460806580395/great_expectations/data_context/data_context/cloud_data_context.py#L277

Reply:
It tries to convert `None` to an empty string and then that fails as well.
```
class GeCloudIdentifier(DataContextKey):
    def __init__(
        self,
        resource_type: "GeCloudRESTResource",
        ge_cloud_id: Optional[str] = None,
        resource_name: Optional[str] = None,
    ) -> None:
        super().__init__()

        self._resource_type = resource_type
        self._ge_cloud_id = ge_cloud_id or ""
        self._resource_name = resource_name or ""
```

Reply:
I created GREAT-1195 to track this, as it is out of scope for this PR.

Reply:
This does seem like a bug we should resolve as part of Thin SaaS though. I agree it is out of scope here but we may want to prioritize as "None" isn't a good look on our end.